src/interval_sets.jl

```diff
@@ -477,8 +477,16 @@ end
 Returns a `Vector{Vector{Int}}` where the value at index `i` gives the indices to all
 intervals in `y` that intersect with `x[i]`.
 """
-find_intersections(x, y) = find_intersections(vcat(x), vcat(y))
-function find_intersections(x::AbstractVector{<:AbstractInterval}, y::AbstractVector{<:AbstractInterval})
+find_intersections(x, y) = find_intersections_(vcat(x), vcat(y))
```

Comment:
Why 1-arg vcat?
```suggestion
find_intersections(x, y) = find_intersections_(collect(x), collect(y))
```

Reply:
To support single intervals in one or both arguments, e.g. `find_intersections([1..2, 4..5], 1.5..4.5)`

Reply:
I'll add a comment to clarify

Reply:
As I'm thinking through this more, I think this is far less useful than I thought it was before 😆, `collect` seems better to me now.